tools/notarize_product.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#https://discuss.atom.io/t/sandbox-supposedly-enabled-but-application-loader-disagrees/26155
+set -o pipefail
+
+if [ ! "${CONFIGURATION}" ]; then
+    CONFIGURATION='Release'
+fi
+
+SCRIPT_DIR="$(dirname "$0")"
+# shellcheck source=tools/ready.sh
+. "${SCRIPT_DIR}/ready.sh" || exit $?
+
+ZIP_PATH=$1/${PACKAGE_NAME}.zip
+PKG_PATH=$1/${PACKAGE_NAME}.pkg
+
+if [ ! "$1" ]; then
+    echo "run archive and put archive path as 1st argument"
+    exit 1
+fi
+
+if [ ! -e "$1" ]; then
+    echo "unexisting path: $1"
+    exit 1
+fi
+
+if [ ! -e "$ZIP_PATH" ] || [ ! -e "$PKG_PATH" ]; then
+    echo "The given path doesn't include .zip or .pkg $ZIP_PATH $PKG_PATH"
+    exit 1
+fi
+
+echo "Apple login ID is required to notarize products"
+echo "Apple ID>"
```

Comment:
`archive_dir`이나 `apple_id`를 받을 때 `echo -n` 하면 개행 문자 없이 출력할 수 있습니다. 이건 취향 따라서 쓰시면 될 것 같네요.
```suggestion
echo -n "Apple ID> "
```